charts/osm/Chart.yaml

```diff
@@ -22,3 +22,9 @@ appVersion: v0.9.1
 
 # This specifies the minimum Kubernetes version OSM is compatible with.
 kubeVersion: ">= 1.19.0"
+
+dependencies:
```

Comment:
~Q: is this always a dependency? (even in situations contour is not going to be deployed)~
Nvm saw the condition flag